DEVELOPING.rst

```diff
@@ -45,7 +45,7 @@ follows:
 
    $ git clone https://github.com/cernopendata/opendata.cern.ch
    $ git clone https://github.com/cernopendata/cernopendata-portal
-   $ cd cernopendata-portal && git checkout -B stable v0.2.9
+   $ cd cernopendata-portal && git checkout -B stable v0.10.2
```

Comment:
Note: Ideally we should do this only from v0.10.3 (or even better v0.11.0 to indicate a breaking change of sorts) when the portal will have port 8000 changes merged.